### DISCUSSION TITLE
[1658280]百英雄传 {本体}

### DISCUSSION THREAD

**Discussion by Temlpe** [pjy612/SteamManifestCache#620] [Share]
### Steam 游戏ID

1658280

### 密钥列表

```hash
1658281;61d12d4ec2b9e888bfe67fae7c70f6d76bfe64bbef403294b7375cc326523bb4
```

### 清单附件

[1658281_4975187447354385205.zip](https://github.com/pjy612/SteamManifestCache/files/15129773/1658281_4975187447354385205.zip)

### 备注

PS：大佬你的那个DepotDownloader[魔改]的项目能更新一下吗，用这个获取清单挺方便的
code:79E3D7BFF9207F05
email:uppuyxw@foxmail.com

**Comment by pjy612** [comment]
DepotDownloader 本质是下载游戏的，目前没啥可更新的。

**Reply by Temlpe** [reply]
前两天DepotDownloader出了新版本感觉可以更一手，好像你的那个魔改版本ssfn授权登录获取清单用不了

**Reply by pjy612** [reply]
是的 之前 ssfn 官方下掉了之后 就不怎么支持的样子。
官方 DepotDownloader 好像就下掉了 ssfn 登陆

**Reply by pjy612** [reply]
。。。官方更新了下 包引用 和 打包相关的。。。然后移除了个括号。。。感觉等于没更。。。
